spec/middlewares/raise_error_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe BaseCRM::Middlewares::RaiseError do
+  before { @raise_error = BaseCRM::Middlewares::RaiseError.new }
+
+  describe "on_complete" do
+
+    it "return nil when code 200..300" do
+      expect(@raise_error.on_complete({:status => 204})).to eql(nil)
+    end
+
+    it "raises ResourceError when code 422 and no content type" do
+      expect { @raise_error.on_complete({:status => 422, :response_headers => {}}) }.to raise_error { |error|
```

Comment:
use `do...end` instead of `{}` for multiline blocks:
```
expect { @raise_error.on_complete({:status => 422, :response_headers => {}}) }
  .to raise_error do |error|
    expect(error).to be_a(BaseCRM::ResourceError)
    expect(error.code).to eql(422)
    expect(error.message).to eql("Unknown error occurred.")
  end
```